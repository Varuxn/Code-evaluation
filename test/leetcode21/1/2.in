100 100
ityihxputrviomkzqkahhytnnmtdnymmjpnegtapw
pservcumxrdhjtocoeoxc
edffkszfyofniahamfveislkdrtkhxnqzzd
ppudoldl
ubqncfbweqjnafv
fspptzyiqemjcfklznmmyliyzjqddjudjzxfpvsahamro
khickxsldjip
xdninizhkjwrfqmkhxegvxjbmgkaiidfldidzckivvttg
pvbmweytchbjboxzimdygvrxfxxlmvfixjxjcad
fuhftfawavtdeycfos
pvfrwmistzkchfhpoygjadoozevvk
vbkfdjvpscgeerz
jjywjiduzcrdndwqttvapgstbndrczrqiermjhlzofhrfzd
wgxkv
tolgijdflenmxfclldmsurfoylnhpnbvegdcoskobjetxkws
plmtzcneqspbmanfcvxvwmxttvbhchehjbnksdsdhtw
vxlpbvcgyiyecwdugbpvcuzzciyfalfjmbeius
liytoxrsgyidveudraltjlepqs
qvnyjatiuvffjirisqzvgephccucjtnzuxkizwiawthokckddf
qebvkccyoanouwj
sesjnhdmabxfhmp
rigpcvwruzlqmjakbmwpisxrrctojgvudsqp
uhspjfbzhevwavaogjpuzh
wn
ukesvblldefadpcjf
rbkmvcevteslnolkazjeqmstqceojvuh
mynoojodojgjanditpslksuxrsg
j
k
raxjzqvnswzqpysiqi
rtebovoujfgsmhdhldlwoumgszfm
vijiejxuczbtxyicbdoonaewnbkdbdpkqmdjdd
zccdrusxkcuctxixcxhkvrozji
tadaacydclpxmsb
xycckpojcxbeycpklqmvlvxj
rstvwxwypyavakfyvxptdkib
yjsdivminz
povkagvlejqohvhjceihwqkfrjrjydnfevyptiupjknieol
fqbxnkeejhr
jbfrzownqyidtlcjghyeigs
xrwktfazlfteteburtpfdbdowjhiqlajkzvwzlzuec
rjoxzstc
zjfvaaqyw
qxieyfpiercobbvxxyrhzelnultxdevzltywhhwwqwxnsexbw
sonwntarmcbynkksasqashoaurbaxrskqmif
mwstvpcexv
aurgnaaonuliezqw
bwozgrw
vaywxofvlqspkqyn
mejjbwkcprnspwnas
aleudogamdlsuqxnlhxzrprjlfhysoifgwpsjqmbohrez
mmhsuxwkesqiachmfkqnwradqhjjfupkjg
tciboupllejgvblhtbbdkvmeksrxifetjqhllrhojstgeo
gyxczrvytiocpmlxirnloxqxvzxkdbokun
cbjrsaobirjwsrbkckcfejbixcklyzlbjbquvhjtpauumwjqcx
meyfafutpvvexvbd
lhpwzdnoyvecjpznlwzkncatyrdun
cpxnqlpktpikbvmmmbkozrauqrxofdzt
eqkqcacxmblmcfdmp
klrgcgpw
nvrwuknunnmdgrghupg
bxxksiddpliscjwqglkmtzklsnzxjunclvsw
jejnrsjaxmzpmkadislwvizkovuavmonjna
hoezgpommnzjiccvuivyctrayabcdrduibpc
tpczsxicmzhsuzipsrfikziwrcefvlvfkssctwsdrgighatcy
jaoywwmijlatnmbpdxnljprigvbhbvpupvugnqhdjxr
jawroisyyjsixllxkwomltscuxzthcwadlcspfg
cahypmxqjgkzgrlyedkrcqrckhnpxgongfzal
edgmuewieibhxrycyfwjplxebhpn
wjllozpqrxfrgehyoigvctrbargvzgtqfqgtluyu
qifddmodcm
rwxubtpvumxwtjvfuqkhgybpfqoljdckym
avwazmdhljbswzacubwcsufyoqtekwazrsnwgjvsp
gpmtkuiswoheuxgfmotoodjadtomr
vcslxvveufqvgfmfng
bfjaqwdjidemykrcvuhndrxtat
xnnzdiwdavebvwcvtymtnrqixpbonaglitklkljvg
klpxfeescmbyvloaxmqsumckm
jksqtpsrmuzeggnxdiucrnkgsnhafsaaizmksrgde
gemouutjilihtqvwqchpbrunbmvlglnxxztfj
zcynvvixfrdcxddqvtaovtvflfbrzklewsowx
sjzonltldreshuzbwphdpprowjzcqwmhyvtfhyzl
obqnktercdq
hcmzfdqucpvwonhyrjmwoigo
ttpzmwluoglzlllgp
qgmgzqeamcqvbrqzlbbgehellaprdmxhvlemahmidyqjcqokg
ajswbcaehkjf
uqxegrfmu
rmwlrdbxtlhpxheruygneeitoysmtwkgsqmzxen
phjnfzxflchxzcxwxkkcbsyjnakvcquqs
xzwywoczojoinimboqqoqoqxhcmeeshuwksfvfkuhdp
uxjofcppyadfvntirpnsqgdpespkvyjijs
lwyufnsskghlptplzbqkhfakxdofchjtvewnbubugkjz
inotggnf
jdgnzxitsqsstdgcotchakwpyjksmzfnmzgzkztjgdn
jyxanenxewosyqicpbidnpbsdoavdyd
jfcvwesnjgoetciikehnkptmwmfdkbtoojtfjt
awnkwnhuj
snfwrrdcfbsnvwkymrcpmthgwaxtmuzf
nmgdiucrzgkxsaignjjeiygtvhgyyvvbckti
sjggsesvrdyausamztonwxzlibbxhwqertmkbzatuwming rigpcvwruzlqmjakbmwpisxrrctojgvudsqpes sonwntarmcbynkksasqashoaurbaxrskqmifqwerqwe jaoywwmijlatnmbpdxnljprigvbhbvpupvugnqhdjxred uxjofcppyadfvntirpnsqgdpespkvyjijsed ksfxoeaviecqlbphepdbhzllglxhjlgusadbeyjszbzvlchues nqdvaxfzcvkhaeqvehotndxqgcling lhhtpygkmoumbdciwcplhdhaoagshbxlmomdzn nmpgkcmyvornwizpbwxofpqfcdqwwcwztsbhifrfnxdjhvring nnjkhtcpenxamjlogtdavymuoxnstfoefqhavheing bvlmekrorrpffsqgqtwxefgkcqyeseeing tlktnjyfed qyxygcrbifenvsejgbfpdawgajognming tbudmjizping fspptzyiqemjcfklznmmyliyzjqddjudjzxfpvsahamroed inotggnfing yeting bwozgrwing szmxajabtzhacvavnmwqjpvlutmluefhkcbfxnclsvungdredred rgzbjchsy bboyivaugrihcssafbizming nwyevoaqaemqxtnptrogawqlnizkyrfheooming ityihxputrviomkzqkahhytnnmtdnymmjpnegtapwes rtebovoujfgsmhdhldlwoumgszfming snfwrrdcfbsnvwkymrcpmthgwaxtmuzfes vaywxofvlqspkqynqwerqwe cpxnqlpktpikbvmmmbkozrauqrxofdztes oing wgxkves tpcufwicdagumuicohrrypkbpaarsgxivhvgagmbcycing fuhftfawavtdeycfosing vogpfmzcimagoxbfavszhphfoxuzzywavdzgscjxfjqing qyrwlpsvezqxjrvnuvxpxing qouglgbjmwgxxhaxing pxhfslmyotkvajxving cpxnqlpktpikbvmmmbkozrauqrxofdztqwerqwe qebvkccyoanouwjing wgxkved htpiuxjmombgafcgiuekoing mffkxxsluqlhzdzrtkued ffqlupdwtdzcsssqpasmsqlahkhtzqpbozpmthnivvfydewing bwacflbjtjgunwohquopvzoewbing zjfvaaqywes jjywjiduzcrdndwqttvapgstbndrczrqiermjhlzofhrfzded mabhhodhgqapglyykusfvnyfdoacyakjwunjmqsed wffugqykilkes hqscgqmpljing xrwktfazlfteteburtpfdbdowjhiqlajkzvwzlzueced fqeanogvhed jdgnzxitsqsstdgcotchakwpyjksmzfnmzgzkztjgdning ityihxputrviomkzqkahhytnnmtdnymmjpnegtapw yasgqqugqydjrvpjqkqtxtfsscbdbmgfblzytgnytizqdvbrdqwerqwe rbkmvcevteslnolkazjeqmstqceojvuhing ztsaubkhmnvtcvcpipiehfqfhing rbhnkvhoiyvpovdokgcndrdqing rstvwxwypyavakfyvxptdkibing qodydqvepvfesixoacked efhjtcrtjyzgetgvprvgkgfslfumemowtknajaryouhkkqj snfwrrdcfbsnvwkymrcpmthgwaxtmuzfes egausmlfpfokroldving zcynvvixfrdcxddqvtaovtvflfbrzklewsowxing ttzvhiqoring uhspjfbzhevwavaogjpuzhed rqoxkgwcixfskzfbmetucrezicoltgidqumes soeoxajkwying uqxegrfmuing hoezgpommnzjiccvuivyctrayabcdrduibpces obqnktercdqed raxjzqvnswzqpysiqiing sesjnhdmabxfhmp jciqrsjznkfyxzmkachroazhxofyw qebvkccyoanouwjing aleudogamdlsuqxnlhxzrprjlfhysoifgwpsjqmbohrezing wgxkvqwerqwe xxxhwlqjnkfimkillkhjvkhoxtking fqbxnkeejhres tgkylelrniflbuqdxnbes ajswbcaehkjfing rstvwxwypyavakfyvxptdkibing nqzwatdjlkieookxxmyyclktdtkfwed cevfolbsmlxneaomsirpqhxejcppdbed cpxnqlpktpikbvmmmbkozrauqrxofdzted sbipzjjiing rigpcvwruzlqmjakbmwpisxrrctojgvudsqping ubqncfbweqjnafving lperthhigpdpdmckudbxzed sekpbntaghrving bfjaqwdjidemykrcvuhndrxtated iougrsqslodpyyztmslmjavmwabwitdwckzhlging gzpoing gtxaqodgwojlqkqjiqohpwjxabxioamoizzpwhplpeolsrwoxzing iih ajswbcaehkjfing wrzbsapqgzxwed aurgnaaonuliezqwed mynoojodojgjanditpslksuxrsges xkhlqkqwerqwe cobzajvflwvczbvgnpzgywyzvinhyupldhampswkpqmeing bzeezkportafgkzgcogfhfpvypvkluyfubonsxfybing xbrddwswvpqmxing
