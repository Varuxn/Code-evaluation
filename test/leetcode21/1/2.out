sjggsesvrdyausamztonwxzlibbxhwqertmkbzatuwming rigpcvwruzlqmjakbmwpisxrrctojgvudsqp sonwntarmcbynkksasqashoaurbaxrskqmif j uxjofcppyadfvntirpnsqgdpespkvyjijs k nqdvaxfzcvkhaeqvehotndxqgcling lhhtpygkmoumbdciwcplhdhaoagshbxlmomdzn nmpgkcmyvornwizpbwxofpqfcdqwwcwztsbhifrfnxdjhvring nnjkhtcpenxamjlogtdavymuoxnstfoefqhavheing bvlmekrorrpffsqgqtwxefgkcqyeseeing tlktnjyfed qyxygcrbifenvsejgbfpdawgajognming tbudmjizping fspptzyiqemjcfklznmmyliyzjqddjudjzxfpvsahamro inotggnf yeting bwozgrw szmxajabtzhacvavnmwqjpvlutmluefhkcbfxnclsvungdredred rgzbjchsy bboyivaugrihcssafbizming nwyevoaqaemqxtnptrogawqlnizkyrfheooming ityihxputrviomkzqkahhytnnmtdnymmjpnegtapw rtebovoujfgsmhdhldlwoumgszfm snfwrrdcfbsnvwkymrcpmthgwaxtmuzf vaywxofvlqspkqyn cpxnqlpktpikbvmmmbkozrauqrxofdzt oing wgxkv tpcufwicdagumuicohrrypkbpaarsgxivhvgagmbcycing fuhftfawavtdeycfos vogpfmzcimagoxbfavszhphfoxuzzywavdzgscjxfjqing qyrwlpsvezqxjrvnuvxpxing qouglgbjmwgxxhaxing pxhfslmyotkvajxving cpxnqlpktpikbvmmmbkozrauqrxofdzt qebvkccyoanouwj wgxkv htpiuxjmombgafcgiuekoing mffkxxsluqlhzdzrtkued ffqlupdwtdzcsssqpasmsqlahkhtzqpbozpmthnivvfydewing bwacflbjtjgunwohquopvzoewbing zjfvaaqyw j mabhhodhgqapglyykusfvnyfdoacyakjwunjmqsed wffugqykilkes hqscgqmpljing xrwktfazlfteteburtpfdbdowjhiqlajkzvwzlzuec fqeanogvhed j ityihxputrviomkzqkahhytnnmtdnymmjpnegtapw yasgqqugqydjrvpjqkqtxtfsscbdbmgfblzytgnytizqdvbrdqwerqwe rbkmvcevteslnolkazjeqmstqceojvuh ztsaubkhmnvtcvcpipiehfqfhing rbhnkvhoiyvpovdokgcndrdqing rstvwxwypyavakfyvxptdkib qodydqvepvfesixoacked efhjtcrtjyzgetgvprvgkgfslfumemowtknajaryouhkkqj snfwrrdcfbsnvwkymrcpmthgwaxtmuzf egausmlfpfokroldving zcynvvixfrdcxddqvtaovtvflfbrzklewsowx ttzvhiqoring uhspjfbzhevwavaogjpuzh rqoxkgwcixfskzfbmetucrezicoltgidqumes soeoxajkwying uqxegrfmu hoezgpommnzjiccvuivyctrayabcdrduibpc obqnktercdq raxjzqvnswzqpysiqi sesjnhdmabxfhmp j qebvkccyoanouwj aleudogamdlsuqxnlhxzrprjlfhysoifgwpsjqmbohrez wgxkv xxxhwlqjnkfimkillkhjvkhoxtking fqbxnkeejhr tgkylelrniflbuqdxnbes ajswbcaehkjf rstvwxwypyavakfyvxptdkib nqzwatdjlkieookxxmyyclktdtkfwed cevfolbsmlxneaomsirpqhxejcppdbed cpxnqlpktpikbvmmmbkozrauqrxofdzt sbipzjjiing rigpcvwruzlqmjakbmwpisxrrctojgvudsqp ubqncfbweqjnafv lperthhigpdpdmckudbxzed sekpbntaghrving bfjaqwdjidemykrcvuhndrxtat iougrsqslodpyyztmslmjavmwabwitdwckzhlging gzpoing gtxaqodgwojlqkqjiqohpwjxabxioamoizzpwhplpeolsrwoxzing iih ajswbcaehkjf wrzbsapqgzxwed aurgnaaonuliezqw mynoojodojgjanditpslksuxrsg xkhlqkqwerqwe cobzajvflwvczbvgnpzgywyzvinhyupldhampswkpqmeing bzeezkportafgkzgcogfhfpvypvkluyfubonsxfybing xbrddwswvpqmxing